analytic_engine/src/compaction/scheduler.rs

```diff
@@ -480,7 +486,8 @@ impl ScheduleWorker {
         waiter_notifier: WaiterNotifier,
         token: MemoryUsageToken,
     ) {
-        let keep_scheduling_compaction = compaction_task.contains_min_level();
+        let keep_scheduling_compaction = self.limit.pending_task_size() < self.max_ongoing_tasks
```

Comment:
I think it is like a condition for judging the degree of pending queue crowding? It seems not very relevant with `max_onging_tasks`? 
How about make it a new config item?

Reply:
fixed